frontend/src/scenes/paths/Paths.scss

```diff
@@ -3,6 +3,7 @@
     max-height: 100%;
     width: 100%;
     height: 720px;
+    overflow: hidden;
```

Comment:
Fixes this

![2023-11-23 18 39 39](https://github.com/PostHog/posthog/assets/1851359/c6561b46-c462-43ae-b0cb-d51faffc9622)